ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -445,6 +455,87 @@ Loader {
         }
     }
 
+    Component {
+        id: deletedMessageComponent
+
+        ColumnLayout {
+
+            RowLayout {
+                id: deletedMessage
+                height: 40
+                Layout.fillWidth: true
+                Layout.topMargin: 8
+                Layout.bottomMargin: 8
+                Layout.leftMargin: 16
+                spacing: 8
+
+                readonly property int smartIconSize: 20
+                readonly property int colorId: Utils.colorIdForPubkey(root.deletedBy)
+                readonly property var messageDetails: StatusMessageDetails {
+                    sender.profileImage {
+                        width: deletedMessage.smartIconSize
+                        height: deletedMessage.smartIconSize
+                        assetSettings: StatusAssetSettings {
+                            width: deletedMessage.smartIconSize
+                            height: deletedMessage.smartIconSize
+                            name: root.deletedByContactIcon || ""
+                            isLetterIdenticon: name === ""
+                            imgIsIdenticon: false
+                            charactersLen: 1
+                            color: Theme.palette.userCustomizationColors[deletedMessage.colorId]
+                            letterSize: 14
+                        }
+
+                        name: root.deletedByContactIcon || ""
+                        pubkey: root.deletedBy
+                        colorId: deletedMessage.colorId
+                        colorHash: root.deletedByContactColorHash
+                        showRing: true
+                    }
+                }
+
+                Rectangle {
+                    width: deletedMessage.height
```

Comment:
This should be either `implicit...` or `Layout.preferred...`, it's wrong to set `width`/`height` for items inside a Layout. 
Same for items below